extension/snippets/dvclive.code-snippets

```diff
@@ -0,0 +1,111 @@
+{
+  "DVCLive Catalyst": {
+    "prefix": "dvclive-catalyst",
+    "body": [
+      "from dvclive.catalyst import DVCLiveCallback",
+      "",
+      "runner.train(",
+      "    model=${1:model}, criterion=${2:criterion}, optimizer=${3:optimizer}, loaders=${4:loaders},",
+      "    callbacks=[DVCLiveCallback()])"
+    ],
+    "description": "DVCLive Catalyst callback"
+  },
+  "DVCLive Fast.ai": {
+    "prefix": "dvclive-fastai",
+    "body": [
+      "from dvclive.fastai import DVCLiveCallback",
+      "",
+      "learn = tabular_learner(${1:data_loader}, metrics=${2:accuracy})",
```

Comment:
Let's not include it, as it is specific to tabular data, and there is no easy way to generalize 

```suggestion
```